cap-ci/config.yaml

```diff
@@ -0,0 +1,5 @@
+{{ define "config" }}
+{
+  "create_k8s": true
```

Comment:
> Why include a parameter to disable creating clusters?
> I understood that we want that to happen always on deployed pipelines.

we need this because one should be able to test things on their own cluster, if required.

Reply:
> Also, if this parameter is false, the pipeline doesn't apply, because the slices `slice "create-k8s" "deploy" …` will point to unexisting jobs..

what do you mean? there are conditions in pipeline definition to handle `false`

Reply:
No, it fails. See my other comment with the snippet to testing this PR.